.travis.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Can you remove (not add) this file that ember-cli helpfully created, we don't use travis..